src/seasidepropertyhandler.cpp

```diff
@@ -54,6 +54,7 @@ void processPhoto(const QVersitProperty &property, bool *alreadyProcessed, QList
     // information in the process of decoding the data...
 
     // Try to interpret the data as a URL
+    *success = false;
```

Comment:
This is a behavioural change; previously it would not be set to false on failure.  Might impact instances where multiple details are converted, but, looks good to me.